docs/src/pages/components/snackbars/IntegrationNotistack.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
-import { SnackbarProvider, VariantType, withSnackbar, withSnackbarProps } from 'notistack';
+import { SnackbarProvider, VariantType, withSnackbar, WithSnackbarProps } from 'notistack';
```

Comment:
Don't know why this passes CI but fails locally. Probably some filesystem-case-sensitive shenanigans.

Reply:
Used a linked package locally.